ebos/eclbasevanguard.hh

```diff
@@ -231,8 +231,8 @@ public:
                              "Include OPM-specific keywords in the ECL restart file to enable restart of OPM simulators from these files");
         EWOMS_REGISTER_PARAM(TypeTag, std::string, IgnoreKeywords,
                              "List of Eclipse keywords which should be ignored. As a ':' separated string.");
-        EWOMS_REGISTER_PARAM(TypeTag, bool, EclStrictParsing,
-                             "Use strict mode for parsing - all errors are collected before the applicaton exists.");
+        EWOMS_REGISTER_PARAM(TypeTag, std::string, ParsingStrictness,
+                             "Set strictness of parsing process. Avaliable options are normal (stop for critical errors), high (stop for all errors) and low (as normal, except do not stop due to unsupported keywords even if marked critical");
```

Comment:
The `"Set strictness ... marked critical"` string literal is 209 characters long, which means the line itself is nudging towards 250 characters.  We might consider splitting the literal across multiple lines to make the source code easier to read for those who don't have wide screens&mdash;or prefer to have more editor windows on the screen.

Reply:
Done.